gcp/appengine/frontend_handlers.py

```diff
@@ -237,102 +237,6 @@ def vulnerability_redirector(potential_vuln_id):
   return None
 
 
-def bug_to_response(bug, detailed=True):
```

Comment:
Moving these functions to `utils.py` so they can be used by both of `api_handlers` and `frontend_handlers`.